k8s_fundamentals/00_setup/setup_cluster.sh

```diff
@@ -8,6 +8,10 @@ fi
 if [[ -z $PROJECT_NAME ]]; then
   echo "INPUT: Type PROJECT_NAME (student-XX-project):" && read PROJECT_NAME
 fi
+
```

Comment:
We should pin the Kubernetes version from our side (and ensure that the tasks are working with this K8s version). Otherwise we will have different versions per attendee which may cause troubles.

Reply:
Since the version provided in script 1.18 was not supported, I thought it would be good idea to have this passed as parameter. and we can instruct the attendees which latest stable version to use such as 1.20.10-gke.10 during training time.  Is it okay?

Reply:
If you did the tasks with 1.20 we can update to this version. But we should not make this customizable per attendee.

Reply:
Taken care, updated the setup.sh script to pin the value of GKE cluster version. Please review!